spec/bowlingcardSpec.js

```diff
@@ -0,0 +1,132 @@
+describe('Bowlingcard', function() {
+  var bowlingcard;
+  beforeEach(function() {
+    bowlingcard = new Bowlingcard();
+  });
+
+  describe('.score', function() {
+    it('starts game at 0', function() {
+      expect(bowlingcard.score).toBe(0);
+    });
+  });
+
+  describe('.enterScore', function() {
+    it('records the users score', function() {
+      bowlingcard.enterScore(5);
+      expect(bowlingcard.score).toBe(5);
+    });
+    
+    it('increase the count of rolls after user enters score for a roll', function() {
+      bowlingcard.enterScore(5);
+      bowlingcard.enterScore(10);
+      expect(bowlingcard.roll).toBe(2);
+    });
+   
+    it('stores the users score in its corresponding roll', function() {
+      bowlingcard.enterScore(5);
+      bowlingcard.enterScore(10);
+      expect(bowlingcard.rollScores).toEqual([5, 10]);
+    })   
+  });
+
+  describe('.roll', function() {
+    it('starts game at 0', function() {
+      expect(bowlingcard.roll).toBe(0);
+    })
+    it('records the users scores as they accumulate', function() {
+      bowlingcard.enterScore(5);
+      bowlingcard.enterScore(10);
+      expect(bowlingcard.score).toBe(15);
+    });
+  })
+
+  describe('.strike', function() {
+    it('records a value or 0 for the following roll if the preceding roll was a strike', function () {
+      bowlingcard.enterScore(10);
+      bowlingcard.isStrike()
+      expect(bowlingcard.rollScores[1]).toEqual(0);
+    })
+  })
+    it('checks if the user rolled a strike when the user enters their score', function() {
+      bowlingcard.enterScore(10);
+      expect(bowlingcard.rollScores[1]).toEqual(0);
+  })
+  it('is triggered only if the roll that scored 10 was an odd roll', function() {
```

Comment:
Now I see why you say these are your rules for bowling ;)
Whenever you work on a project, it's worth talking to your client (or here, maybe to peers) until the requirements are clear. If needed, write a number of input/output examples to see if you got it right.